src/test/java/com/seam/api/connectwebview/ConnectWebviewsTest.java

```diff
@@ -0,0 +1,55 @@
+package com.seam.api.connectwebview;
+
+import com.seam.api.Seam;
+import com.seam.api.TestUtils;
+import com.seam.api.resources.connectwebviews.requests.ConnectWebviewsCreateRequest;
+import com.seam.api.resources.connectwebviews.requests.ConnectWebviewsGetRequest;
+import com.seam.api.types.ConnectWebview;
+import com.seam.api.types.ConnectWebviewsCreateRequestAcceptedProvidersItem;
+import com.seam.api.types.ConnectWebviewsCreateRequestProviderCategory;
+import java.util.List;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public final class ConnectWebviewsTest {
+
+    private static Seam seam;
+
+    @BeforeAll
+    public static void beforeAll() {
+        seam = TestUtils.startFakeSeam(8080);
+    }
+
+    @Test
+    public void test_connect_webviews() throws InterruptedException {
+        ConnectWebview createdWebView = seam.connectWebviews()
+                .create(ConnectWebviewsCreateRequest.builder()
+                        .acceptedProviders(List.of(ConnectWebviewsCreateRequestAcceptedProvidersItem.SCHLAGE))
+                        .build());
+        Assertions.assertThat(createdWebView.getConnectWebviewId()).isNotNull();
+
+        ConnectWebview webview = seam.connectWebviews()
+                .get(ConnectWebviewsGetRequest.builder()
+                        .connectWebviewId(createdWebView.getConnectWebviewId())
+                        .build());
```

Comment:
Very interesting, can we have an alias here?

```suggestion
                .getById(createdWebView.getConnectWebviewId());
```

Reply:
Yup, definitely can. I'll add the alias to all resources.